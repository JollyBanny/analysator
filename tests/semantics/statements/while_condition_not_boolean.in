var i : integer = 0;

begin
    while i + 1 do begin
        write(i);
        i += 1;
    end;
end.