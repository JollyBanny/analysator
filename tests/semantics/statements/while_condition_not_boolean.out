[Line: 4; Column: 13]	boolean expected but integer found
